staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics/metrics.go

```diff
@@ -146,6 +146,14 @@ var (
 		},
 		[]string{"resource"},
 	)
+
+	WatchCacheReadWait = compbasemetrics.NewHistogram(
+		&compbasemetrics.HistogramOpts{
+			Name:           "apiserver_watch_cache_read_wait_seconds",
```

Comment:
Use Namespace and Subsystem and then Name of read_wait_seconds

Reply:
fixed, thanks!